psi4/driver/procrouting/sapt/sapt_jk_terms.py

```diff
@@ -389,6 +389,140 @@ def induction(cache, jk, do_print=True, maxiter=12, conv=1.e-8, do_response=True
             core.print_out(print_sapt_var(name, ret[name], short=True))
             core.print_out("\n")
 
+    # Exch-Ind without S^2
+    if Sinf:
+        nocc_A = cache["Cocc_A"].shape[1]
+        nocc_B = cache["Cocc_B"].shape[1]
+        SAB = core.Matrix.triplet(
+            cache["Cocc_A"], cache["S"], cache["Cocc_B"], True, False, False)
+        num_occ = nocc_A + nocc_B
+
+        Sab = core.Matrix(num_occ, num_occ)
+        Sab.np[:nocc_A, nocc_A:] = SAB.np
+        Sab.np[nocc_A:, :nocc_A] = SAB.np.T
+        Sab.np[np.diag_indices_from(Sab.np)] += 1
+        Sab.power(-1.0, 1.e-14)
+
+        Tmo_AA = core.Matrix.from_array(Sab.np[:nocc_A, :nocc_A])
+        Tmo_BB = core.Matrix.from_array(Sab.np[nocc_A:, nocc_A:])
+        Tmo_AB = core.Matrix.from_array(Sab.np[:nocc_A, nocc_A:])
+
+        T_A = np.dot(cache["Cocc_A"], Tmo_AA).dot(cache["Cocc_A"].np.T)
+        T_B = np.dot(cache["Cocc_B"], Tmo_BB).dot(cache["Cocc_B"].np.T)
+        T_AB = np.dot(cache["Cocc_A"], Tmo_AB).dot(cache["Cocc_B"].np.T)
+
+        T_A = core.Matrix.from_array(T_A)
+        T_B = core.Matrix.from_array(T_B)
+        T_AB = core.Matrix.from_array(T_AB)
+
+        sT_A = np.dot(cache["Cvir_A"], unc_x_B_MOA.transpose()).dot(Tmo_AA).dot(cache["Cocc_A"].np.T)
+        sT_B = np.dot(cache["Cvir_B"], unc_x_A_MOB.transpose()).dot(Tmo_BB).dot(cache["Cocc_B"].np.T)
+        sT_AB = np.dot(cache["Cvir_A"], unc_x_B_MOA.transpose()).dot(Tmo_AB).dot(cache["Cocc_B"].np.T)
+        sT_BA = np.dot(cache["Cvir_B"], unc_x_A_MOB.transpose()).dot(Tmo_AB.np.T).dot(cache["Cocc_A"].np.T)
+
+        sT_A = core.Matrix.from_array(sT_A)
+        sT_B = core.Matrix.from_array(sT_B)
+        sT_AB = core.Matrix.from_array(sT_AB)
+        sT_BA = core.Matrix.from_array(sT_BA)
+        
+        jk.C_clear()
+    
+        jk.C_left_add(core.Matrix.chain_dot(cache["Cocc_A"], Tmo_AA))
+        jk.C_right_add(cache["Cocc_A"])
+    
+        jk.C_left_add(core.Matrix.chain_dot(cache["Cocc_B"], Tmo_BB))
+        jk.C_right_add(cache["Cocc_B"])
+    
+        jk.C_left_add(core.Matrix.chain_dot(cache["Cocc_A"], Tmo_AB))
+        jk.C_right_add(cache["Cocc_B"])
+    
+        jk.compute()
+    
+        J_AA, J_BB, J_AB = jk.J()
+        K_AA, K_BB, K_AB = jk.K()
+
+        # A <- B
+        EX_AA = V_B.clone()
```

Comment:
Can you add an `_inf` to the names of the tensors so it is very clear that they differ from the S^2 tensors.